src/platform/intel/cavs/include/cavs/lib/memory.h

```diff
@@ -35,13 +38,6 @@
 
 #define MAX_MEMORY_SEGMENTS		PLATFORM_HPSRAM_SEGMENTS
 
-#if CONFIG_LP_MEMORY_BANKS
-#define LP_SRAM_SIZE \
-	(CONFIG_LP_MEMORY_BANKS * SRAM_BANK_SIZE)
-#else
-#define LP_SRAM_SIZE 0
-#endif
-
```

Comment:
This will break XTOS build of Intel cAVS platforms.

I'm trying an alternative at https://github.com/thesofproject/sof/pull/6678  -- let's see if it goes through the checks.